pkg/controller/postgresqluser/postgresqluser_controller.go

```diff
@@ -361,6 +387,41 @@ func (r *ReconcilePostgreSQLUser) groupByHosts(hosts HostAccess, namespace strin
 	return errs
 }
 
+// groupAllDatabasesByHost groups read write accesses for all known databases in the hosts access map.
+func (r *ReconcilePostgreSQLUser) groupAllDatabasesByHost(hosts HostAccess, host string, namespace string, access lunarwayv1alpha1.AccessSpec, privilege postgres.Privilege) error {
+	databases, err := r.allDatabases(r.client, namespace)
+	if err != nil {
+		return fmt.Errorf("get all databases: %w", err)
+	}
+	var errs error
+	for _, databaseResource := range databases {
+		database := databaseResource.Spec.Name
+		schema := databaseResource.Spec.Name // FIXME: This will not work when schemas differ from database names
```

Comment:
FIXME is still here

Reply:
Yeah, it's one of the limitations of this change set.

I'm not sure how to handle it, maybe you have an idea?

When using the `allDatabases` flag we do not specify the schemas so it could be that we should grant access to all databases and all schemas in them.
The only way to do that would be to query the database after available schemas and grant access to each of them.

This imposes an issue when a schema is created by a service, ie. it is not declared in k8s. How can we know something changed?

Reply:
I think it is a fair requirement that a service just works inside it's defined `schema`.
Maybe just remove the `FIXME` and then add it as a comment so that we are aware of the issue if we run into it later on.

Reply:
We unfortunately cannot use this with some of our services that uses the `public` schema. But fair enough. I'll make it a comment for now.